windows/src/global/inc/Compfile.h

```diff
@@ -79,7 +79,7 @@ struct FILE_STORE {
   BOOL fIsOption;
   BOOL fIsDebug;
   BOOL fIsCall;
-  int line;
+  int line; // TODO: int vs dword, line vs Line (see FILE_KEY, FILE_GROUP)
```

Comment:
Do you miss updating this, to be DWORD?

Reply:
I added it as a future TODO. It's one of those little things -- casing is easy enough, but changing the type needs to be checked carefully for boundary conditions, so didn't want to put that into this PR.